addons/hr_timesheet/views/project_task_portal_templates.xml

```diff
@@ -73,6 +73,26 @@
                 </t>
             </td>
         </xpath>
+        <xpath expr="//tbody[@t-if='tasks']/tr/th" position="after">
```

Comment:
I think this will cause issues with the "Documents" button, which is placed at the end of the first `th`

Reply:
I named the `th` in the module project so I can xpath it more precisely now